src/common/libschedutil/init.h

```diff
@@ -0,0 +1,26 @@
+/************************************************************\
+ * Copyright 2019 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef _FLUX_SCHEDUTIL_INIT_H
+#define _FLUX_SCHEDUTIL_INIT_H
+
+struct schedutil_ctx {
+    flux_t *h;
+    struct ops_context *ops_ctx;
+};
+typedef struct schedutil_ctx schedutil_t;
```

Comment:
I'd probably drop the typedef and go with `struct schedutil` here.  The typedef doesn't add much, especially since the structure def is public.

Reply:
In a later commit, I refactored the struct defintion to no longer be public.  Sorry for that confusion.  Based on other discussions in this thread, if I make the struct definition public again, I'll drop the typedef.